integrations/getUserProfile/index.test.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from "node-fetch";
+
+import { get as getEnvValue } from "../../lib/env";
+
+describe("getUserProfile", () => {
```

Comment:
I like to use `operationId`s because they refer to OpenAPI identifiers for each endpoint specification. I consider the actual url of the endpoint an implementation detail, while the actual name is more meaningful for our domain. 
Also, I plan to automate over the spec file in next stages of this suite, and `operationId` would be better reference than actual paths.

if you think it's not adding any issue, I'd leave this.